tests/pytests/unit/utils/test_json.py

```diff
@@ -0,0 +1,107 @@
+"""
+Tests for salt.utils.json
+"""
+
+import textwrap
+
+import pytest
+
+import salt.utils.json
+
+
+def test_find_json():
+    some_junk_text = textwrap.dedent(
+        """
+        Just some junk text
```

Comment:
In the description of your PR you stated the extra data that was getting added was errors and warnings. Does this mean `[DEBUG]` or `[WARNING]` could be in the raw data? If so can we also add a test with a debug or warning log and ensure it parses correctly? Any other expected odd text that could be added that we can add a test to ensure it parses correctly?

Reply:
Actually the input strinc could contain a line starting with `[WARNING]` before of after JSON, but it's also possible that it could be any message with no `[WARNING]` tag, but agree it makes sense to add such test cases.

Reply:
Added two more test cases